packedForest/src/forestTypes/binnedTree/processingNode.h

```diff
@@ -51,6 +51,7 @@ namespace fp{
 
 					int tempSwap;
 
+                                        // This is a way to sample without replacement the number of features for mtry.
```

Comment:
sorry.. not the number of features... it's sampling the features themselves (w/o replacement).  also, mentioning the shuffling maybe also a good idea... (sorry I asked you to take that out)